github/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "../tsconfig.base.json",
+    "compilerOptions": {
+        "outDir": "out",
+        "rootDir": ".",
```

Comment:
We should turn on the declaration option here unless we have a good reason not to. That way we can remove `index.d.ts` and we don't have to maintain the types later on.

Reply:
Noticed you made carried these changes over already, thanks for doing that!